fp_app/app/controllers/appointments_controller.rb

```diff
@@ -0,0 +1,63 @@
+class AppointmentsController < ApplicationController
+  before_action :authenticate_user!, only: %i[create update]
+  before_action :correct_user, only: %i[create update]
+
+  def create
+    ActiveRecord::Base.transaction do
+      @appointment = Appointment.new(
+        user_id: params[:user_id],
+        planner_id: params[:planner_id],
+        schedule_id: params[:schedule_id],
+        reserved_at: params[:reserved_at],
+        status: "reserved"
+      )
+
+      unless @appointment.save
+        logger.error "Appointment save failed: #{@appointment.errors.full_messages.join(', ')}"
+        raise "Failed to create appointment"
+      end
```

Comment:
```suggestion
      @appointment.save!
```
`!` をつければActiveRecord側がraiseしてくれます（何か意図があってこの形であればそのままでも良いです）

Reply:
ありがとうございます！
` @appointment.save!`の方がシンプルに書けるので変更しておきます。